node/src/BaseClient.ts

```diff
@@ -3015,6 +3016,36 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Returns the number of members in the sorted set stored at <code>key</code> with scores between
+     * 'minLex' and 'maxLex'.
+     *
+     * See https://valkey.io/commands/zlexcount/ for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param minLex - The minimum lex to count from. Can be positive/negative infinity, or specific lex and inclusivity.
+     * @param maxLex - The maximum lex to count up to. Can be positive/negative infinity, or specific lex and inclusivity.
+     * @returns The number of members in the specified lex range.
+     * - If 'key' does not exist, it is treated as an empty sorted set, and the command returns '0'.
+     * - If maxLex less than minLex, '0' is returned.
+     *
+     * @example
+     * ```typescript
+     * const result = await client.zlexcount("my_sorted_set", {value: "c"}, 'positiveInfinity');
+     * console.log(result); // Output: 2 - Indicates that there are 2 members with lex scores between "c" (inclusive) and positive infinity in the sorted set "my_sorted_set".
+     *
+     * const result = await client.zlexcount("my_sorted_set", {value: "c"}, {value: "k", isInclusive: false});
+     * console.log(result); // Output: 1 - Indicates that there is one member with LexBoundary "c" <= score < "k" in the sorted set "my_sorted_set".
```

Comment:
Kinda makes it seem like LexBoundary is a class, maybe we should change to align with the previous example:
```suggestion
     * console.log(result); // Output: 1 - Indicates that there is one member with a lex score between "c" (inclusive) and "k" (exclusive) in the sorted set "my_sorted_set".
```